src/Indicator.vala

```diff
@@ -68,6 +107,11 @@ public class DateTime.Indicator : Wingpanel.Indicator {
                 idle_update_events ();
             });
 
+            cal_button.clicked.connect (() => {
+                calendar.open_maya ();
```

Comment:
This seems to be the only call to this method, so why is it in another class instead of in this class?